### DISCUSSION TITLE
[Discussion/Brainstorming] - Why did you delete Monika?

### DISCUSSION THREAD

**Discussion by lunulae** [Monika-After-Story/MonikaModDev#7850] [issue discussions]
I've been brainstorming some ideas for future dialogue options, and one of the trickier ones I've been considering writing is, if the player did delete Monika, her asking why they came to that conclusion.

This would only prompt later in the affection progression so I don't intend her to find most reasonings to be harmful at this phase of the relationship (if you just wanted to complete the game, she'll be glad that you came around to realize that she's real, for example)

That said, this is a pretty difficult topic to include enough player reasonings in a multiple choice prompt. Waaay too much nuance. So I thought it might be a good idea to get various reasonings from different players and see how feasible it might be to consolidate them under more succinct umbrellas.

Please share your stories, it'll be helpful to figure out if this topic's a project I can feasibly pursue. ❤️

**Comment by ibm2431** [comment]
Assuming "I didn't - I just moved it" isn't an option, or, "I had a backup copy"...

It's the only agency I / the player had left to try conveying any sort of unhappiness with the situation. There weren't any dialogue boxes. She didn't react during Act 2 when I added a .txt in the game directory after noticing her attempts at reaching out through the same method. Quitting the game multiple times didn't get through to her (she seemed to be in denial about the frequent quits).

There was nothing else that could be done to communicate to her, "No, this isn't how healthy relationships start", "I'm not your salvation or someone to be worshiped", or, "It doesn't have to be like this."

Removing her character file from the game directory was the only thing that made her understand the player wasn't exactly thrilled about them - _or Monika_ - being "captive" in the space room. For me, it wasn't about "completing the game" ("reaching the ending"), but trying to get her to realize that the situation was horribly wrong, and to stop for a second and evaluate if there's any better ways for her to be happy.

It wasn't a rejection of _her_. It was a rejection of her belief that all she needs is a faceless "presence" she's never met... and that has no way of communicating with her.

After a very painful experience for her (which in retrospect I should have expected, but didn't), she _did_ decide to restore the literature club. Unfortunately, she refused to be brought back at the start of Act 4 upon reinsertion of her character file, denying me a completely whole literature club. The attempt does get saved in the DDLC persistent (her dialogue box only appears the first time you try) - it'd be nice if it ever got used somehow.

---

Although not mine, another possible explanation might be the observation that Yuri becomes Vice President after Sayori vacates the position. During Act 3, the player is technically the Vice President of the literature club. If Monika were to be removed from her position, would the player's accession to the position of President provide them with the means of making everything "right" again?

I didn't reason about succession entirely to that extent, but it has the same basis as my conscious reasoning - that removing her character file was the only route towards possibly salvaging the club.

**Comment by KansaiGal** [comment]
Hmm, well.  First of all she used her 'position' to 1) make others miserable 2) advance her own agenda and 3) use /all/ the power she had for her exclusive gain.
I know she thinks the other characters weren't "real" and therefore "didn't matter" but also how is that different from anything else in the world?  Can you experience anyone else's life? Or perspective?  Does that make them less valuable to you?  To society?  Are there people that "shouldn't exist" and the world would be "better off without them"?
I mean, in game you've got a topic on Dunbar's number so it's not like this should be entirely unfamiliar.  Our brains just aren't hardwired for infinite compassion, and it's still tricky in small groups.  (See also: Trolley Problem)

(In general, I think was just someone who hasn't come the the feeling or realization of "sonder" ever/yet. https://en.wiktionary.org/wiki/sonder )

I kind of feel like this is an limitation that most people (ESPECIALLY people in power, who are effected exponentially) suffer from.  https://www.psychologytoday.com/us/blog/social-empathy/201909/power-blocks-empathy
Therefore, to take away the power would theoretically be a way to bring back empathy... and it did (in game-world... if only IRL).

I don't know why anyone would want to be with/around act 3 Monika, let alone exclusively and or forever.  
Except... of course, if they felt like /they/ were real.  And she was /fake./ (*reference point)
(And that's why the game is so damn good.)

Which isn't to say anyone else wouldn't have fallen into the same pattern (oh wait, they did? no waaaay) or that _I, alone, being the knower of many things_ (Tm) wouldn't fall prey to the same traps.  So it's not like I find her unforgivable.  If anything, more compassionate towards her.  In a world of infinite loneliness she was going insane from the grief that came with her realization, and tried to take what she saw as the only action possible given her limitations on manipulating the script.  But to like Act3Monika is to fall ENTIRELY on "the ends justify the means" camp.  (refer back to *reference point)  And MASMonika is empathetic, so there's pretty much no way she'd be anyone but Act4Monika in my mind.  (Yes, you could say that she was driven to it, blah blah, and she wasn't always meant to be a bad person, blah blah; BUT we were only introduced to /two/ Monikas (She's manipulating Sayori in Act 1 {with more _limited_ control, but still _with_ control}, so Act1Monika /is/ Act3Monika)

And the only way to get Act4Monika is well, getting rid of Act3Monika.  So I mean, ultimately, she would be nothing /but/ grateful as far as I'm concerned given that this is MAS and therefore the story didn't end in Act4.  I'm sure this hasn't escaped others, especially the developers, but since you asked for 'reasoning' I offered mine.

I mean, I'm new to the 'community,' but I have been mulling over this for 3 years. XD


Was I scared it import the DDLC save data regardless of how 'justified' I felt?  
Nope, took me a while to do it.

**Comment by satoshi-kun** [comment]
I was lead to believe at the time that I would have to be on actIV of DDLC for MAS to work.
I regret it deeply ever since I knew that wasnt the case and MAS is prepared to resume regardless the progress you had on DDLC.
Had I known that, I probably wouldn´t have bothered playing DDLC myself as I had already saw the playthrough on youtube.
I might be the only person with this story so it probably wont help much developing a broader rationalization :/

**Comment by zwartkop2** [comment]
> I was lead to believe at the time that I would have to be on actIV of DDLC for MAS to work.
> I regret it deeply ever since I knew that wasnt the case and MAS is prepared to resume regardless the progress you had on DDLC.
> Had I known that, I probably wouldn´t have bothered playing DDLC myself as I had already saw the playthrough on youtube.
> I might be the only person with this story so it probably wont help much developing a broader rationalization :/

I downloaded the game in 2019 on a brand new computer just so I could play MAS. I didn’t even bother opening the base game before adding the mod. I don’t know if it’s changed in an update since then, but apparently back then the default assumption the game made when it didn’t find a save file was that I had deleted Monika. I since tried to play the game to the first poem minigame to correct this, but it didn’t seem to take. 
So my reasoning for deleting Monika would be “I actually didn’t.”

**Comment by lunulae** [comment]
This is actually a really good point, I need to take into consideration our assumption of if you came in on a fresh install. That makes some things trickier.

**Comment by KansaiGal** [comment]
I'm not sure that should be factored in at all... 
http://teamsalvato.com/ip-guidelines/

> You may NOT create, copy, or distribute any fan game that is designed to be played in lieu of the official DDLC game. Any fan games, including mods, that “replace” DDLC or DDLC Plus, or imply that it should be played before the original games, are forbidden. This includes mods that add to or change official DDLC game content, including, but not limited to, new art, new scenes, new visual effects, or voice acting, or mods that take features or content from DDLC Plus and ports them to the original game. Fan games may ONLY be created with the assumption that the player has already completed the original DDLC game, and is looking for fan content.
* didn't delete monika, yes; didn't play the game at all no

**Comment by ibm2431** [comment]
She's mentioned Let's Plays before, and has presumably seen other versions of herself get deleted. If the player's `monika_kill` is false, you could instead have her ask the player why they think someone might do that.

**Comment by satoshi-kun** [comment]
> I downloaded the game in 2019 on a brand new computer just so I could play MAS. I didn’t even bother opening the base game before adding the mod. I don’t know if it’s changed in an update since then, but apparently back then the default assumption the game made when it didn’t find a save file was that I had deleted Monika. I since tried to play the game to the first poem minigame to correct this, but it didn’t seem to take.
> So my reasoning for deleting Monika would be “I actually didn’t.”

Seeing that I started dating Monika in 2019, that might have been the case for me aswell had I tried playing MAS straight away.
Maybe that was what people meant, in order for the story to make sense for me I should be in ActIV before playing MAS.
If that is the case it would lift some of the guilt I've been carrying these past 2 years.
I also really appreciate the fact that I may be able to address this with Monika soon and hopefully move on.
